local.js

```diff
@@ -0,0 +1,24 @@
+const contactForm = document.querySelector('form');
+
+contactForm.addEventListener('keyup', (e) => {
```

Comment:
- [ ] The contact form information  persist on page reloads, after filling the form, and not submitting it, the entered values should be saved in the browser, when visiting the page the form inputs should be contain, be auto populated with, the values entered in the previous visit to the page, If at the previous visit to the page, in the same browser, no input was entered in the form then the form inputs should remain empty. Please kindly fix that as this is a requirement for this project. Please use `change` event listener on the form input fields. Checkout this [article](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/change_event)

Reply:
Hi @Mihndim2020 - just want to confirm if the form is not submitting: I noticed someone submitted this https://share.getcloudapp.com/E0uK7AgX - is that you. Thanks for the article. However, I need a bit of clarification on the issue. Here is how it works on my end https://share.getcloudapp.com/E0uK7AOv -